packages/valory/skills/abstract_round_abci/behaviour_utils.py

```diff
@@ -1631,7 +1634,11 @@ def get_transaction_receipt(
         :yield: LedgerApiMessage object
         :return: transaction receipt data
         """
-        self._send_transaction_receipt_request(tx_digest, retry_timeout, retry_attempts)
+        if chain_id is None:
+            chain_id = self.params.default_chain_id
+        self._send_transaction_receipt_request(
+            tx_digest, retry_timeout, retry_attempts, chain_id=chain_id
+        )
```

Comment:
@Adamantios I looked for the methods where we're interacting with ledger/contract, `_send_transaction_receipt_request` was the only one where we could not specify the `chain_id`, if there are any other please let me know